src/libraries/System.IO/tests/StreamWriter/StreamWriter.StringCtorTests.cs

```diff
@@ -29,8 +32,10 @@ public static void EmptyPath_ThrowsArgumentException()
         {
             // No argument name for the empty path exception
             AssertExtensions.Throws<ArgumentException>(null, () => new StreamWriter(""));
+            AssertExtensions.Throws<ArgumentException>(null, () => new StreamWriter("", new FileStreamOptions()));
             AssertExtensions.Throws<ArgumentException>(null, () => new StreamWriter("", true));
             AssertExtensions.Throws<ArgumentException>(null, () => new StreamWriter("", true, Encoding.UTF8));
+            AssertExtensions.Throws<ArgumentException>(null, () => new StreamWriter("", Encoding.UTF8, new FileStreamOptions()));
```

Comment:
I only see negative tests here for StreamReader/Writer.  Are there tests that validate the ctors do what they're supposed to do when used correctly?

Reply:
I fully agree they are missing, but I did not find any 'positive' test covering the existing constructors with path, FileMode, FileAccess, FileShare, bufferSize and FileOptions parameters.
Which type of tests do you expect here?

Reply:
> Which type of tests do you expect here?

I was primarily thinking about tests to validate that we're correctly passing through the arguments, e.g. that if you pass CustomEncoding as the encoding, that's actually what gets used by the writer.